app/json/slate_lineup_configs.json

```diff
@@ -550,69 +514,15 @@
   },
   {
     "id": "05027beb-0053-4020-8bdc-4da2fcc0cb68",
```

Comment:
I see a [reference to this id in web-qa](https://github.com/Pocket/web-qa/blob/242705ab9e69c2939ecc4fff18f32405a7fa46e2/scripts/generators/seed.js#L33), so I'm inclined to keep this lineup around for now.